packages/model-runner/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@covid-modeling/model-runner",
-  "version": "0.8.0",
+  "version": "0.9.0",
```

Already on 1.0.0, just forgot to bump here.
```suggestion
  "version": "1.1.0",
```